src/user/UserRouter.ts

```diff
@@ -133,6 +134,13 @@ router.delete('/api/1.0/users/:id', tokenAuthentication, async (req: Request, re
     }
 
     await UserService.deleteUser(req.params.id);
```

Comment:
Will now delete every related Token when user is deleted because of relationship and onDelete cascade